prisma/seed.ts

```diff
@@ -0,0 +1,72 @@
+import { PrismaClient } from '@prisma/client';
+
+const prisma = new PrismaClient();
+
+async function main() {
+  const password = 'bitsofgood';
```

Comment:
@Arthelon Do we want to use a default password for all seed data or let the developers specify it from the environment variable?